packages/protocol-kit/src/utils/signatures/utils.ts

```diff
@@ -1,11 +1,16 @@
+import { ethers } from 'ethers'
 import {
   EthAdapter,
   SafeSignature,
-  SafeTransactionEIP712Args
+  SafeEIP712Args,
+  SafeTransactionData
 } from '@safe-global/safe-core-sdk-types'
 import { bufferToHex, ecrecover, pubToAddress } from 'ethereumjs-util'
+import semverSatisfies from 'semver/functions/satisfies'
 import { sameString } from '../address'
 import { EthSafeSignature } from './SafeSignature'
+import { getEip712MessageTypes, getEip712TxTypes } from '../eip-712'
+import { SigningMethod } from '../..'
```

Comment:
Could we set a route here that is easier to understand?

`@safe-global/protocol-kit` or `@safe-global/protocol-kit/types`